docs/patterns/retry-content.md

```diff
@@ -4,7 +4,7 @@ Enable an application to handle transient failures when it tries to connect to a
 
 An application that communicates with elements running in the cloud has to be sensitive to the transient faults that can occur in this environment. Faults include the momentary loss of network connectivity to components and services, the temporary unavailability of a service, or timeouts that occur when a service is busy.
 
-These faults are typically self-correcting, and if the action that triggered a fault is repeated after a suitable delay it's likely to be successful. For example, a database service that's processing a large number of concurrent requests can implement a throttling strategy that temporarily rejects any further requests until its workload has eased. An application trying to access the database might fail to connect, but if it tries again after a delay it might succeed.
+These faults are typically self-correcting, and if the action that triggered a fault is repeated after a suitable delay it's likely to be successful. For example, a database service that's processing a large number of concurrent requests can implement a [throttling strategy](throttling) that temporarily rejects any further requests until its workload has eased. An application trying to access the database might fail to connect, but if it tries again after a delay it might succeed.
```

Comment:
this link doesn't look right to me `(throttling)` -- can you please make them linked like you did on the others?

Reply:
Since the throttling page is under the same directory as the retry page, I thought that this would be the correct relative link for it. Added a commit now that will hopefully be better.

Thanks.

Reply:
I commited the update, it's `./throttling.yml`